checker/bin-devel/test-qualifier-consistency.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
```

Comment:
Please add some comments here about the assumptions the script makes, e.g. where it assumes the JDK.